skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -99,37 +103,47 @@ export default class search extends Component {
 
   Validate() {
     let Valid = false;
-    if( parseInt(this.state.score) < 1 || this.state.score === null || this.state.score === "" || (this.state.score % 1) !== 0) 
-    {
+    if (parseInt(this.state.score) < 1 || this.state.score === null 
+    || (this.state.score % 1) !== 0) {
```

Comment:
I mean if score is string, convert it so you can check for mod validation in your case `this.state.score % 1`
as well as check for empty string like you were doing it previously

Reply:
I dont allow for string inputs

Reply:
Is Score string or Int?

Reply:
its an int . The input type is set to number